package.json

```diff
@@ -44,7 +45,10 @@
     "compile": "tsc -p . && cp -r src/v2 build/src/ && cp -r proto* build/ && cp test/*.js build/test",
     "prepare": "npm run compile",
     "pretest": "npm run compile",
-    "proto": "mkdir -p proto && pbjs -t static-module -w commonjs -p protos google/bigtable/v2/bigtable.proto | pbts -o proto/bigtable.d.ts -"
+    "proto": "npm run proto:bigtable && npm run proto:bigtable_instance_admin && npm run proto:bigtable_table_admin",
+    "proto:bigtable": "mkdir -p proto && pbjs -t static-module -w commonjs -p protos google/bigtable/v2/bigtable.proto | pbts -i long -o proto/bigtable.d.ts -",
+    "proto:bigtable_instance_admin": "mkdir -p proto && pbjs -t static-module -w commonjs -p node_modules/google-proto-files google/bigtable/admin/v2/bigtable_instance_admin.proto | pbts -i long -o proto/bigtable_instance_admin.d.ts -",
```

Comment:
Any reason why we are building from node_modules instead of the protos directory?

Reply:
Hm, yeah good question and no excuse for you to be honest. I was keeping the script consistent across Datastore and Spanner and didn't realize we were sourcing the latter two files differently. Lesson learned - don't copy, paste, forget 😅 Fixed!